BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -564,11 +564,12 @@ extension ScrollerWrapperView: KeyboardObserverDelegate {
     func keyboardFrameWillChange(
         for observer: KeyboardObserver,
         animationDuration: Double,
-        options: UIView.AnimationOptions
+        animationCurve: UIView.AnimationCurve
     ) {
-        UIView.animate(withDuration: animationDuration, delay: 0.0, options: options, animations: {
+        UIViewPropertyAnimator(duration: animationDuration, curve: animationCurve) {
```

Comment:
We could stick with `UIView.animate` - we'd just need to do a `UIView.AnimationOptions(rawValue: animationCurve << 16)`.